frontend/src/App.tsx

```diff
@@ -48,6 +49,15 @@ const App: React.FC = () => {
               </ProtectedRoute>
             )}
           />
+
+          <Route
+            path="/seedlot-details"
```

Comment:
Change the path here, 'seedlot' should be a starting point for the rest of the URL, like 'seedlot/detail'.